babycenter/templates/search/search_results.html

```diff
@@ -5,31 +5,43 @@
 {% block content %}
 {% if search_results %}
   <div class="search">
```

Comment:
Remove this container, and rename the h3, to h1, and give it a classes of `heading heading--orange`